pom.xml

```diff
@@ -68,6 +68,29 @@
           </markdown>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.cyclonedx</groupId>
+        <artifactId>cyclonedx-maven-plugin</artifactId>
+        <configuration>
```

Comment:
This belongs to the main, because the goal of SBOM is get all child dependencies from maven considering all directories.

The parameters are set also on the configuration to exclude the directories that does not belongs to production.